.github/workflows/ci.yaml

```diff
@@ -29,6 +29,13 @@ jobs:
         uses: ./ci/images/debian10
         with:
           args: ./ci/steps/test.sh
+  e2e:
```

Comment:
note to self: it may be redudant to have a step for testing and end to end. since we'll use jest as the test runner for e2e tests, we can probably just make this work in one job